packages/connect-node/src/node-universal-client.ts

```diff
@@ -280,7 +280,7 @@ function h2Request(
   options: Omit<http2.ClientSessionRequestOptions, "signal">,
   onStream: (stream: http2.ClientHttp2Stream) => void,
 ): void {
-  const requestUrl = new URL(url, sm.authority);
+  const requestUrl = new URL(url);
```

Comment:
The URL constructor parses `https:localhost:8443`, but not when the `base` argument is given.

The result here used to be `https://localhost:8443/localhost:8443/connectrpc.eliza.v1.ElizaService/Introduce`, and is now the expected `https://localhost:8443/connectrpc.eliza.v1.ElizaService/Introduce`.

IIRC, the authority was only given as a base to potentially support relative URLs, but I don't actually see us supporting them in a meaningful way with this HTTP client.